middle_end/printclambda_primitives.ml

```diff
@@ -96,6 +96,18 @@ let primitive ppf (prim:Clambda_primitives.primitive) =
       in
       let name = "make" ^ mode ^ mut in
       fprintf ppf "%s %i%a" name tag Printlambda.block_shape shape
+  | Pmakeufloatblock(mut, mode) ->
+      let mode = match mode with
```

Comment:
I actually did this originally and then reverted it because:
- This file also exists in the `ocaml` subtree, and it seems worth keeping the two copies the same.
- In that other file, this would be refactoring of existing code creating diffs against upstream and potential rebasing conflicts.
But obviously both choices kind of suck.  If after considering that you feel it's clearly better to have the nice code and deal with the diffs, I'm happy to change it.

Reply:
Yes, that makes sense. Feel free to resolve similar comments elsewhere.